charmcraft/commands/init.py

```diff
@@ -18,13 +18,24 @@
 
 import logging
 import os
-import pwd
 import re
+import sys
 from datetime import date
+from typing import Optional
 
 from charmcraft.cmdbase import BaseCommand, CommandError
 from charmcraft.utils import make_executable, get_templates_environment
 
+try:
+    import ctypes
+except ImportError:
+    ctypes = None
```

Comment:
What this is for? Is there any platform where `ctypes` does not exist?

Reply:
Ah yes, I forgot. I'm only used to seeing it on Windows :D

Reply:
Done